test/test_helper.rb

```diff
@@ -39,12 +39,19 @@ def assert_has_component_title(title)
     end
   end
 
-  def assert_has_component_govspeak(content)
-    within shared_component_selector("govspeak") do
+  def assert_has_component_govspeak(content, index: 1)
```

Comment:
Nice pattern. I think we could re-use this pattern on one of the stats announcements which has multiple components of the same type, metadata I think. cc @dsingleton